aleph-client/src/connections.rs

```diff
@@ -24,14 +24,17 @@ use crate::{
 /// Capable of communicating with a live Aleph chain.
 #[derive(Clone)]
 pub struct Connection {
-    client: SubxtClient,
+    /// inner subxt type
+    pub client: SubxtClient,
 }
 
 /// Any connection that is signed by some key.
 #[derive(Clone)]
 pub struct SignedConnection {
-    connection: Connection,
```

Comment:
Why can't you use `signed_connection.as_connection()`, `signed_connection.signer()` and `connection.as_client()` ?
https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/src/connections.rs#L220
https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/src/connections.rs#L175
https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/src/connections.rs#L387